var/spack/repos/builtin/packages/py-pytorch-lightning/package.py

```diff
@@ -34,21 +35,26 @@ class PyPytorchLightning(PythonPackage):
     version("1.3.8", sha256="60b0a3e464d394864dae4c8d251afa7aa453644a19bb7672f5ee400343cdf7b0")
     version("1.2.10", sha256="2d8365e30ded0c20e73ce6e5b6028478ae460b8fd33727df2275666df005a301")
 
+    # src/pytorch_lightning/__setup__.py
```

Comment:
I definitely appreciate the addition of comments like this and the one below.  I think it can help educate ppl on where to find dependencies.